### ISSUE TITLE
Suggestion: Mystcraft Lecterns and Bookstands

### ISSUE THREAD

**Issue by shadowfacts** [tterrag1098/WAILAPlugins#35]
The name of the contained book should be displayed in the WAILA tooltip for the Lecterns and Bookstands from Mystcraft. This would make it a lot easier to find a certain book if the player has a large number of them.

**Comment by tterrag1098** [comment]
I will implement this prior to 1.0 (aka soon)

**Comment by XCompWiz** [comment]
Query: Did this implementation happen? Anything I can do to facilitate it?

**Comment by tterrag1098** [comment]
No, never did. Honestly this mod kind of fell off the wagon, I need to bring it back into shape. Maybe writing this plugin will be my first step :)

**Comment by XCompWiz** [comment]
I know the feeling on falling off the wagon! :D Good luck, and starting with something relatively minor feels like the best bet; so do that, whatever it is. :)
